src/app/core/services/kafka.service.ts

```diff
@@ -121,6 +121,11 @@ export class KafkaService {
   }
 
   prepareKafkaObjectAndSend(task, value, type) {
+    this.firebaseAnalytics.logEvent('prepared_kafka_object', {
+      name: task.name,
+      questionnaire_timestamp: String(task.timestamp),
+      type: type
```

Comment:
Why do we need the type ? Can it not be inferred from the name ?
We should try to keep the parameter to the minimum as there are restrictions of how many can be displayed on the firebase console

Reply:
Type refers to the kind of data, whether it is a completion_log, timezone, or assessment. The name refers to the task name associated with it (PHQ8, etc).

Reply:
ok cool